snapcraft/plugins/python.py

```diff
@@ -36,7 +36,8 @@
       Path to a constraints file
     - process-dependency-links:
       (bool; default: false)
-      Enable the processing of dependency links.
+      Enable the processing of dependency links in pip, which allow one
```

Comment:
You wouldn't say "dependency links allows one project to provide …", you'd say "dependency links allow one project to provide…"